packages/runtime/src/middleware/error-handling.ts

```
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from "express";
+import logger from "../util/logger";
+import { serializeError } from "serialize-error";
+import { BaseError, HttpStatusCode } from "../errors";
+
+export default (error: any, request: Request, response: Response, next: NextFunction) => {
+  if (error.isIntentional) {
```

this means that if an app dev throws an `Error` instead of a `BaseError` (or derived type) then we'll translate it to a 500?

---

Yup, they need to throw an error that we know about